docs/source/models.rst

```diff
@@ -914,6 +914,32 @@ Furthermore, if you want to run model inference in the same environment used in
 from the `conda.yaml` file, ensuring that the python UDF will execute with the exact package versions that were used
 during training.
 
+Some PyFunc models may accept inference configuration, which controls how the model is loaded and inference
+performed. You can learn which inference configuration the model supports by inspecting the model's flavor metadata:
+
+.. code-block:: python
+
+    model_info = mlflow.models.get_model_info(model_uri)
+    model_info.flavors[mlflow.pyfunc.FLAVOR_NAME][mlflow.pyfunc.INFERENCE_CONFIG]
+
+Alternatively, you can load the PyFunc model and inspect the `inference_config` property:
+
+.. code-block:: python
+
+    pyfunc_model = mlflow.pyfunc.load_model(model_uri)
+    pyfunc_model.inference_config
+
+Inference configuration can be changed at loading time by indicating `inference_config` parameter in the 
+:py:func:`mlflow.pyfunc.load_model` method:
```

Comment:
Can we rename `inference_config` to `load_config` or `load_params`?

Reply:
I the original proposal we used the name `params` which is close to `load_params`. However, we thought that would be confusing with the `params` at predict time. I completely see your point cause this configuration may be used for inference but more generally to load the model. If I have to vote, I would rather use `config` but `params` also works. I would remove the `load` word because it will be redundant when you read it at `mlflow.pyfunc.load_model(load_config=...)`. Which one do you think conveys better the idea?

Reply:
I changing it to `model_config`. I think it plays well with `model_uri`... `model_config` while we don't use the same `params` word we use for predict.